style(edit-content): improve actions layout and single tab mode

oidacra opened the pull request:
- Organize action buttons into left/right groups to prevent layout jumps
- Fix single tab mode border and padding styles
- Add consistent spacing and alignment for action items

This pull request includes several changes to the `dot-edit-content-form` component in both its HTML and SCSS files. The changes focus on improving the layout and organization of the content actions and tab view.

### Improvements to HTML structure:

* Added a `scrollable="false"` attribute to the `dot-edit-content-tabview` element to disable scrolling.
* Reorganized the content actions by introducing `dot-edit-content-actions__left` and `dot-edit-content-actions__right` divs to better separate lock controls from other actions.
* Closed an unclosed div and added a new div to properly structure the content actions.

### Enhancements to SCSS styling:

* Updated `.dot-edit-content-actions` to align items center, set width to 100%, and define a minimum height.
* Added styles for `.dot-edit-content-actions__left` and `.dot-edit-content-actions__right` to properly align and space elements within these sections.
* Modified the `p-tabview-nav` and `p-tabview-nav-content` classes to ensure proper alignment and border styling within the tab view. [[1]](diffhunk://#diff-1a3e657016880498977ce74202425946ae817683630dd29a8b673b2dd4672e5aR93-R95) [[2]](diffhunk://#diff-1a3e657016880498977ce74202425946ae817683630dd29a8b673b2dd4672e5aR116) [[3]](diffhunk://#diff-1a3e657016880498977ce74202425946ae817683630dd29a8b673b2dd4672e5aL114-R152)

### Screenshots

https://github.com/user-attachments/assets/a40c0d44-b4be-4e51-b2ce-fa49c3bbcb20



This PR fixes: #31783

oidacra committed:
style(edit-content): improve actions layout and single tab mode

- Organize action buttons into left/right groups to prevent layout jumps
- Fix single tab mode border and padding styles
- Add consistent spacing and alignment for action items

nicobytes commented on core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.scss:
```diff
@@ -37,9 +37,23 @@ $tab-min-height: 52px;
 .dot-edit-content-actions {
     display: flex;
-    flex-wrap: wrap;
+    align-items: center;
```
I think we should use more prime flex utilities instead of adding more css classes, in this case:

```
.dot-edit-content-actions {
    min-height: $tab-min-height;
}

<div class="dot-edit-content-actions flex align-items-center w-full">
```

oidacra replied:
✅

nicobytes commented on core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.scss:
```diff
@@ -37,9 +37,23 @@ $tab-min-height: 52px;
+    min-height: $tab-min-height;
+}
+
+.dot-edit-content-actions__left {
```
I think we should use more prime flex utilities instead of adding more css classes, in this case:

```
.dot-edit-content-actions__left {
    min-width: 200px; // Adjust this value based on your needs
}

<div class="dot-edit-content-actions__left flex align-items-center gap-4">
```

oidacra replied:
✅

nicobytes commented on core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.scss:
```diff
@@ -37,9 +37,23 @@ $tab-min-height: 52px;
+}
+
+.dot-edit-content-actions__right {
+    display: flex;
```
I think we should use more prime flex utilities instead of adding more css classes, in this case:

```css
<div class="flex align-items-center gap-4 ml-auto">
```

oidacra committed:
style(edit-content): enhance action buttons layout with consistent flex alignment

- Updated HTML and SCSS for action buttons to ensure consistent flex alignment across left and right groups.
- Improved styling for sidebar toggle and action items to enhance overall layout stability.
